aws/sdk/examples/using_native_tls_instead_of_rustls/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "using_native_tls_instead_of_rustls"
+version = "0.1.0"
+authors = ["Zelda Hessler zhessler@amazon.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# aws-config pulls in rustls and several other things by default. We have to disable defaults in order to use native-tls
+# and then manually bring the other defaults back
+aws-config = { path = "../../build/aws-sdk/sdk/aws-config", default-features = false, features = ["default-provider", "native-tls", "rt-tokio", "dns", "tcp-connector"] }
+# aws-sdk-s3 brings in rustls by default so we disable that in order to use native-tls only
+aws-sdk-s3 = { package = "aws-sdk-s3", path = "../../build/aws-sdk/sdk/s3", default-features = false, features = ["native-tls"] }
+# aws-sdk-sts is the same as aws-sdk-s3
+aws-sdk-sts = { package = "aws-sdk-sts", path = "../../build/aws-sdk/sdk/sts", default-features = false, features = ["native-tls"] }
```

Comment:
Thanks to John's suggestion of removing default features for runtime crates, we don't even need to include them here!